src/interface/src/app/treatments/map-project-areas/map-project-areas.component.html

```diff
@@ -37,8 +37,8 @@
   [sourceLayer]="layers.projectAreaLabels.sourceLayer"
   [layout]="{
     visibility: visible && withFill ? 'visible' : 'none',
-    'text-field': ['get', 'rank'],
-    'text-size': 14,
+    'text-field': ['get', 'name'],
+    'text-size': textSize$?.value,
```

Comment:
this is weird, shouldn't this have an `| async` pipe? or are you actually just checking the current value?
if that's the case....then this shouldn't be an observable ?

Reply:
Updated following your suggestion below. Thx!